install/bundle/install.sh

```diff
@@ -14,13 +14,20 @@ getAptPackage(){
     export DEBIAN_FRONTEND=noninteractive
     (echo "mysql-server mysql-server/root_password password $_database_password_";echo "mysql-server mysql-server/root_password_again password $_database_password_") | debconf-set-selections
     #Update apt sources and install
-    dpkg -s gnupg 2>/dev/null || (apt-get update && apt-get install -y gnupg)
-    echo "deb http://ppa.launchpad.net/stesie/libv8/ubuntu bionic main" | tee /etc/apt/sources.list.d/stesie-libv8.list && apt-key adv --keyserver keyserver.ubuntu.com --recv-keys D858A0DF
-    apt-get update && apt-get install -y vim ntp zip unzip curl wget apache2 libapache2-mod-xsendfile libapache2-mod-php php php-dev php-pear php-zip php-mysql php-mbstring mysql-server cmake fp-compiler re2c libv8-7.5-dev libyaml-dev python python3 python3-requests openjdk-8-jdk openjdk-11-jdk
+    add-apt-repository ppa:stesie/libv8 -y
+    echo "deb http://ppa.launchpad.net/stesie/libv8/ubuntu bionic main" | tee /etc/apt/sources.list.d/stesie-libv8.list
+    add-apt-repository ppa:ondrej/php -y
+    find /etc/apt/sources.list.d/ -type f -name "*.list" -exec  sed  -i.bak -r  's#deb(-src)?\s*http(s)?://ppa.launchpad.net#deb\1 http\2://launchpad.proxy.ustclug.org#ig' {} \;
+    find /etc/apt/sources.list.d/ -type f -name "*.list" -exec  sed  -i.bak -r  's#deb(-src)?\s*http(s)?://ppa.launchpadcontent.net#deb\1 http\2://launchpad.proxy.ustclug.org#ig' {} \;
+    apt-get update
+    apt-get install -y libv8 php7.4 php7.4-yaml php7.4-xml php7.4-dev php7.4-zip php7.4-mysql php7.4-mbstring
+    apt-get install -y libseccomp-dev vim ntp zip unzip curl wget libapache2-mod-xsendfile mysql-server php-pear cmake fp-compiler re2c libv8-7.5-dev libyaml-dev python2.7 python3 python3-requests openjdk-8-jdk openjdk-11-jdk openjdk-17-jdk
```

Comment:
```suggestion
    apt-get install -y libseccomp-dev git vim ntp zip unzip curl wget libapache2-mod-xsendfile mysql-server php-pear cmake fp-compiler re2c libv8-7.5-dev libyaml-dev python2.7 python3 python3-requests openjdk-8-jdk openjdk-11-jdk openjdk-17-jdk
```

Docker 镜像中似乎并不自带 Git，可能需要另行安装。

Reply:
此处应该先独立安装git以后才能git clone该repo

Reply:
> 此处应该先独立安装git以后才能git clone该repo

Docker 镜像的构建过程不能使用外部环境的程序。

Reply:
因为看之前master分支的Dockerfile是先安装了git然后git clone repo的，但是这里的部署思路是先git clone再部署，所以没有安装git

Reply:
合并该请求